metadash/config/config.py

```diff
@@ -10,6 +10,7 @@
 
 
 class ConfigItemModel(db.Model):
+    __tablename__ = 'config_item'
```

Comment:
How about add a prefix like __metadash so anyone examining the database would know this table is created by metadash?

Reply:
Done